Meadowlark-js/backends/meadowlark-mongodb-backend/docker/docker-compose.yml

```diff
@@ -0,0 +1,129 @@
+version: '3'
```

Comment:
https://stackoverflow.com/questions/47615751/docker-compose-run-a-script-after-container-has-started

has some good ideas about automating this... in the future. Maybe not worth digging into right now.